content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -221,7 +221,7 @@ A structural schema is an [OpenAPI v3.0 validation schema](#validation) which:
 
 Non-structural example 1:
 
-```none
+```yaml
```

Comment:
This snippet is not a valid yaml due to the trailing `...`.
Please check your other modifications as well.

Reply:
Looks each snippet has a `...` at beginning or end, so it's not necessary to change the code symbol `none`, right?

If yes, this PR can be directly closed. 😺 That's all right.